plugins/modules/fusion_nig.py

```diff
@@ -117,9 +117,6 @@
 from ansible_collections.purestorage.fusion.plugins.module_utils.startup import (
     setup_fusion,
 )
-from ansible_collections.purestorage.fusion.plugins.module_utils.getters import (
-    get_az,
```

Comment:
I wonder if this should go the other way, i.e. whether get_az and get_region checks should be added to make this module consistent with the other ones

Reply:
Nevermind, the general direction is to remove them.

Reply:
I don't think we should call any other APIs in the modules (except where it is required). Not only it adds additional logic to the module, but it also makes more http requests that wouldn't be needed. Also, if you insert incorrect `availability_zone`, you'll get roughly the same error (the API will tell you).